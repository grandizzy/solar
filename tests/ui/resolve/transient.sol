contract A {
    public transient tempValue; //~ ERROR: expected contract item (function, variable, struct, or modifier definition), found keyword `public`
}
