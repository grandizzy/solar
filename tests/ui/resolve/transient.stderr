error: expected contract item (function, variable, struct, or modifier definition), found keyword `public`
  --> ROOT/tests/ui/resolve/transient.sol:LL:CC
   |
LL |     public transient tempValue;
   |     ^^^^^^
   |
   = note: for a full list of valid contract items, see <https://docs.soliditylang.org/en/latest/grammar.html#a4.SolidityParser.contractBodyElement>

error: aborting due to 1 previous error

